css/src/components/hero.scss

```diff
@@ -143,13 +189,25 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 	&.hero-image:dir(rtl) {
 		direction: ltr;
 
-		.hero-content {
+		.hero-content,
+		.hero-details {
 			direction: rtl;
 
 			@include desktop {
 				padding-inline-end: unset;
 				padding-inline-start: $hero-padding-sm;
 			}
 		}
+
+		&.hero-with-details {
```

Comment:
It seems like we have a conundrum here. 

Ideally, we want to avoid shifting the image at all. With this change, the image is flipped and moved to the left. However, it does make sense to read the hero content first before moving onto the hero details. 

![image](https://github.com/microsoft/atlas-design/assets/94572161/40f7bbe4-1030-49b8-9cb9-9a576aee59a6)

If we don't move the image, the hero details will be read first before the hero content, which does seem out of order.

![image](https://github.com/microsoft/atlas-design/assets/94572161/c3362e03-ab9c-4211-affe-9c11e283b59c)

Reply:
I think i would need to ask an RTL user to understand what an expectation here should be.

The code no longer flips this image, its just aligns end with t he content.  SO it appears different in your view, but youre just seeing the right side of the image vs left.